server/src/controllers/tagController.js

```diff
@@ -1,6 +1,15 @@
 import Tag from '../models/tagModel';
 
-// Triggered from '/api/tags'
+// Triggered from 'GET /api/tags'
+const findAllTags = (req, res) => {
+
+  Tag.findAll()
+  .then(tags => res.send(tags))
+  .catch(err => res.send('No teams were found', err));
```

Comment:
no teams were found is a weird response to not being able to find tags? Maybe I am misunderstanding something